src/validation.cairo

```diff
@@ -91,11 +96,8 @@ fn validate_timestamp(self: @ChainState, block: @Block) -> Result<(), ByteArray>
 }
 
 fn next_prev_timestamps(self: @ChainState, block: @Block) -> Span<u32> {
-    let mut prev_timestamps = *self.prev_timestamps;
-    prev_timestamps.pop_front().unwrap(); //keep only 10 most recent previous timestamps
-    let mut timestamps: Array<u32> = array![*block.header.time];
-    timestamps.append_span(prev_timestamps);
-    timestamps.span()
+    // TODO: implement
```

Comment:
Looks like  merge/rebase mistake.

Reply:
this is because the function signature is different now, differente parameters.